Source/monster.h

```diff
@@ -317,6 +317,29 @@ struct Monster { // note: missing field _mAFNum
 		return pgettext("monster", data().name);
 	}
 
+	/**
+	 * @brief Calculates monster's experience points.
+	 * Fetches base exp value from @p MonstersData array.

```

Comment:
```cpp
	 * Fetches base exp value from @p MonstersData array.
```
what's `@p` ?

Reply:
`@p` makes the word after it appear in typewriter font